third_party/move/move-compiler-v2/tests/checking/specs/structs_err.exp

```diff
@@ -1,6 +1,6 @@
 
 Diagnostics:
-error: field `xx` not declared in struct `S`
+error: field `xx` not declared in `S`
```

Comment:
It would be nice if we could at least clarify that this is a Struct or Variant.

Reply:
Done